src/mpid/ch4/netmod/ofi/ofi_capability_sets.h

```diff
@@ -45,7 +46,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_get_set_number(char *set_name)
     } else if (!strcmp("verbs", set_name)) {
         return MPIDI_OFI_SET_NUMBER_VERBS;
     } else {
-        return -1;
+        return MPIDI_OFI_SET_NUMBER_UNKNOWN;
```

Comment:
Should we be consistent with the naming and have this be `MINIMAL` as well as below?